src/Database/Pivot.php

```diff
@@ -1,17 +1,24 @@
 <?php namespace Winter\Storm\Database;
 
 use Illuminate\Database\Eloquent\Model as ModelBase;
-use Illuminate\Database\Eloquent\Builder as BuilderBase;
+use Winter\Storm\Support\Str;
 
 class Pivot extends Model
 {
     /**
      * The parent model of the relationship.
      *
-     * @var \Illuminate\Database\Eloquent\Model
+     * @var \Illuminate\Database\Eloquent\Model|null
      */
     protected $parent;
```

Comment:
Why was this property retained instead of replaced as I did in https://github.com/wintercms/storm/pull/65?

Reply:
I couldn't discount that `$parent` might be used by our users. The edits I made to the Pivot keep it backwards compatible.

Reply:
@bennothommo we should be able to use the AsPivot trait instead in our model and just handle the BC for the parent property by defining getParentAttribute() & setParentAttribute() methods. Thoughts?